tests/commonUtils.sh

```diff
@@ -15,8 +15,8 @@
 #   limitations under the License.
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-RUNTIME_IMAGE='docker.io/istio/runtime:2017-03-22-21.21.48'
-
+MANAGER_HUB_TAG='docker.io/istio,2017-03-22-21.21.48'
```

Comment:
Yes, but aren't both hardcoding a version info? For local integration testing, don't we need to be able to compile manager/mixer (which creates a local docker.io/istio/manager, docker.io/istio/mixer), and then the scripts use these images?